.github/ISSUE_TEMPLATE/good_first_issue.yml

```diff
@@ -42,6 +42,7 @@ body:
           discussions, guides.
       value: |
           - [Contribution guide - start here!](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
+          - [Intel DevHub Discord channel](https://discord.gg/wPuqAujS) - engage in discussions, ask questions and talk to OpenVINO developers
```

Comment:
Let us have a separate server for OpenVINO in Discord. How much will it cost to create the server?
I see minuses in using DevHub Discord:
- create structure of channels with dedicated topics (FEs, core, APIs, announcement, sync-ups with external contributors). 
- mix with other Intel products that is not good. OV has own great reputation in software community, popularity, etc.
- we can control how to access a server, etc. on our own

Reply:
> mix with other Intel products that is not good. OV has own great reputation in software community, popularity. So we are able to collect community on our own.

It seems that the only products present on Intel DevHub Discord (in terms of dedicated channels) are Intel Developer Cloud and OpenVINO.

> we can't create clear (without mix) separate structure of OV channels with dedicated topics (FEs, core, APIs, announcement, sync-ups with external contributors, OVMS, tokenizers, and other openvino toolkit products).

We kind of can, but it wouldn't be as clear as we would probably wish - currently everything's under "Developer Topics" ribbon, but we could add an "OpenVINO" ribbon. However, if/when it comes to that, I think that it would be preferable to just create a separate Discord server, dedicated to OpenVINO.

> Other FWs (TensorFlow, PyTorch, MindSpore) have own servers on Slack, Discord. I think we should have the same level.

I agree, however since the beginning of GFI we've been following the route of growing accordingly - we usually start small and invest more time into it only when there's a clear hint that it's a good direction. 

Today is the first "official" day of OpenVINO developer presence on Discord and we've already had 3 discussions. I would prefer to have a separate OpenVINO Discord server as well, but this would be a risk in terms of time invested and changing our approach, which has worked for us great up until this point.

Reply:
Let us not discuss it here and have it offline. It is not late to change the approach and create a separate server.
I don't know why the decision has not been discussed with internal developers and who is a decision makers.

Reply:
Let us proceed with this as a temporal solution and see how community there is growing